core/examples/luigi-sample-angular/src/index.html

```diff
@@ -8,9 +8,59 @@
     />
     <link rel="stylesheet" href="/luigi-core/luigi.css" />
   </head>
+  <style>
+    /* LUIGI CUSTOM CONTAINER FEATURE */
+    /* #my-header {
+      width: 100%;
+      height: 200px;
+      border: 1px solid;
+    }
+
+    #my-leftnav {
+      position: relative;
+      width: 20%;
+      height: 500px;
+      border: 1px solid;
+      float: left;
+    }
+
+    [luigi-app-root] {
+      float: left;
+      width: 70%;
+      height: 500px;
+    }
+
+    #my-rightnav {
+      position: relative;
+      width: 10%;
+      height: 500px;
+      border: 1px solid;
+      float: right;
+    }
+
+    #my-footer {
+      position: relative;
+      width: 100%;
+      height: 300px;
+      border: 1px solid;
+    } 
+    
+    .custom {
+      padding: 20px;
+      text-align: center;
+    } */
+  </style>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <script src="/luigi-core/luigi.js"></script>
     <script src="/assets/extendedConfiguration.bundle.js"></script>
+
+    <!-- LUIGI CUSTOM CONTAINER FEATURE -->
+    <!-- <div class="custom" id="my-header">MY HEADER</div> 
+    <div class="custom" id="my-leftnav">MY LEFTNAV</div>
+    <div luigi-app-root></div>
+    <div class="custom" id="my-rightnav">MY RIGHTNAV</div>
+    <div style="clear: both"></div>
```

Comment:
If you are fine with it, I would suggest to restructure it a little bit:

- remove `<div style="clear: both"></div>`
- add  extra wrapper so that the 3 columns are in one `div`
  
<img width="430" alt="wrapper" src="https://user-images.githubusercontent.com/46446373/62229437-eeeac700-b3bf-11e9-9bbe-05ced45a2711.png">

- add styles
   `.wr {
      position: relative;
      overflow: hidden;
   }`

Reply:
I also think it is better as you proposed it. Done.